llama-index-integrations/llms/llama-index-llms-oci-genai/llama_index/llms/oci_genai/base.py

```diff
@@ -322,26 +306,164 @@ def gen() -> ChatResponseGen:
 
         return gen()
 
-    async def acomplete(
-        self, prompt: str, formatted: bool = False, **kwargs: Any
-    ) -> CompletionResponse:
-        # do synchronous complete for now
-        return self.complete(prompt, formatted=formatted, **kwargs)
-
     async def achat(
         self, messages: Sequence[ChatMessage], **kwargs: Any
     ) -> ChatResponse:
-        # do synchronous chat for now
-        return self.chat(messages, **kwargs)
+        raise NotImplementedError("Async chat is not implemented yet.")
```

Comment:
There's no async client methods we can implement?

Reply:
@logan-markewich Our Python SDK doesn't have async support natively. We probably need to re-evaluation how we gonna implement async. Probably will be another PR as well.